### ISSUE TITLE
Publish build for electron 7.1

### ISSUE THREAD

**Issue by pgherveou** [grpc/grpc-node#1235]
Is there a build published for Electron 7.1 ? 

npm fallbacks to compiling code when using latest stable (7.1.8)
```
node-pre-gyp WARN Using needle for node-pre-gyp https download 
node-pre-gyp WARN Tried to download(404): https://node-precompiled-binaries.grpc.io/grpc/v1.24.2/electron-v7.1-darwin-x64-unknown.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for grpc@1.24.2 and electron@7.1.7 (electron-v7.1 ABI, unknown) (falling back to source compile with node-gyp) 
```

**Comment by murgatroid99** [comment]
We're having some build problems. When we get it working again we will publish binaries for Electron 7.1.

**Comment by pgherveou** [comment]
Thanks for the update! Good luck fixing it 🙏🏻

**Comment by josiahruddell** [comment]
Any update on this?

**Comment by murgatroid99** [comment]
The build problems turned out to be more complicated than expected. We're actively working on fixing them but it may still take some time.

**Comment by sstone1** [comment]
@murgatroid99 is there any update to this issue? Are the build problems still stopping the publishing of 1.25.0?

**Comment by murgatroid99** [comment]
Builds for Electron 7 and above are available in the latest version 1.24.4.
